composer.json

```
@@ -78,7 +82,8 @@
 		"rmccue/requests": "^1.8.1",
 		"wp-coding-standards/wpcs": "2.*",
 		"wp-phpunit/wp-phpunit": "~6.0",
-		"yoast/phpunit-polyfills": "^1.1"
+		"yoast/phpunit-polyfills": "^1.1",
+		"wporg/wporg-parent-2021": "dev-build"
```

These are sorted alphabetically (or they should be, I see the `require` is not _quite_ right).

```suggestion
		"wporg/wporg-parent-2021": "dev-build",
		"yoast/phpunit-polyfills": "^1.1"

```